src/terms/ewald.jl

```diff
@@ -82,56 +82,45 @@ function energy_ewald(lattice, recip_lattice, charges, positions; η=nothing, fo
     max_exp_arg = -log(eps(T)) + 5  # add some wiggle room
     max_erfc_arg = sqrt(max_exp_arg)  # erfc(x) ~= exp(-x^2)/(sqrt(π)x) for large x
 
+    # Precomputing summation bounds from cutoffs.
+    # Let A be the lattice matrix, then if ||Ax|| <= R, 
+    # then xi = <ei, A^-1 Ax> = <A^-T ei, Ax> <= ||A^-T ei|| R.
+    #
+    # Reciprocal space:  ||A^-1 G|| / 2η ≤ max_erfc_arg
```

Comment:
with A the reciprocal lattice

Reply:
I took the sqrt on both sides of the inequality, replacing max_exp_arg by max_erfc_arg; maybe squares would be more natural though?

I now renamed the recip lattice in the comment as B.

Those in the comments are already the expressions we were bounding IIUC ?

Reply:
Ah I see, but this is a bit "by accident" as `max_erfc_arg` only happens to be `sqrt(max_exp_arg)` because of the way we approximate it. I think explicit is better here.

Yeah, what I meant is that similar to the ti, tj thing it is not obvious to a code reader at that point why these are the expressions that matter. I think just explain this (stating that collected terms are of the form exp(- stuff) for real space and erfc(stuff) for reciprocal.